src/app/ui/editor/brush_preview.cpp

```diff
@@ -165,11 +163,14 @@ void BrushPreview::show(const gfx::Point& screenPos)
 
   tools::Tool* tool = m_editor->getCurrentEditorTool();
   const bool isFloodfill = tool->getPointShape(0)->isFloodFill();
-  // TODO add support for "tile-brushes"
+
+  // Get current tilemap mode
+  TilemapMode tilemapMode = site.tilemapMode();
```

Comment:
A little nice change is to put this as `const`:
```suggestion
  const TilemapMode tilemapMode = site.tilemapMode();
```
This indicates that the `tilemapMode` is used just like a constant value/an alias of `site.tilemapMode()`. (There are a lot of missing `const` in the code, so there is no problem if this is added or not).